python/lsst/pipe/base/__init__.py

```diff
@@ -15,3 +15,4 @@
 from . import connectionTypes
 from . import pipelineIR
 from .executionButlerBuilder import *
+from .status import *
```

Comment:
I had not realized I imported them directly from `pipe.base.status` in ctrl_mpexec; I think that was carelessness, and I'll go fix that.

And yes, that does mean that `_status` may be a better name, and if daf_butler I'd definitely have called it that. I have been hesitant to start using leading-underscore modules in other packages because it's so much less common outside daf_butler and the hierarchy is usually much simpler, but I'm happy to start doing it here (and more often in general) if you prefer.  I did RFC it as a practice we _can_ do a while back.

Reply:
I was going to say I think we should move that way, but really I don't care, do what you think is best. If this was in a sub module I would say do it, but really whatever you want.

I actually kind of liked it being imported as status in ctrl mpexec, because it made it feel more like an enum or some other grouping, and some way to figure out what other ones were, but again, no strong feeling.